config/200-webhook-clusterrole.yaml

```diff
@@ -0,0 +1,28 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: knative-eventing-webhook
+rules:
+  # For watching logging configuration and getting certs
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get", "list", "watch"]
+  # For manipulating certs into secrets
+  - apiGroups: [""]
+    resources: ["secrets"]
```

Comment:
I wonder if this is a candidate for a (namespaced) `Role` so the webhook doesn't have access to all secrets everywhere. Seems like maybe configmaps and deployments permissions could also be namespaced.

Reply:
ok to make this into a separate issue and tackle that in a followup?